src/node/desktop/package.json

```diff
@@ -66,11 +66,14 @@
     "typescript": "4.5.2"
   },
   "dependencies": {
+    "@types/bindings": "^1.5.1",
```

Comment:
@matheustavaresdev recommends we pin specific versions so we can decide when to update a package and try it out very carefully... So no "^" in front.